docs/_config.yml

```diff
@@ -1,4 +1,4 @@
-title: "HCA Ingest Developer Docs"
+title: "HCA Ingest Documentation"
```

Comment:
Do the docs state the last update date somewhere? 
Idealy, each article would show its updated date.